### PR TITLE
feat: change cancerRelated headername into inknownledgebasegene

### PR DESCRIPTION
**Pull request by kttkjl**
just header name changed for now
DEVSU-1125

### CODE REVIEW

File: app/views/ReportView/components/CancerPredisposition/columnDefs.ts [hunk 1]
```diff
@@ -39,15 +39,11 @@ const columnDefs: ColDef[] = [{
...
-      return `(${
-        variant.gene1.name || '?'
-      },${
-        variant.gene2.name || '?'
-      }):fusion(e.${
-        variant.exon1 || '?'
-      },e.${
-        variant.exon2 || '?'
-      })`;
+      return `(${variant.gene1.name || '?'
```
Comment (areisle) [inline app/views/ReportView/components/CancerPredisposition/columnDefs.ts:42]: this change seems to be style only and doesn't seem related to the PR or the commit message. was this just autofixed or something like that?

File: app/views/ReportView/components/SmallMutations/columnDefs.ts [hunk 2]
```diff
@@ -17,11 +17,10 @@ const columnDefs: ColDef[] = [{
   hide: false,
 }, {
   headerName: 'Location',
-  valueGetter: ({ data }) => data.chromosome && `${data.chromosome}:${data.startPosition}${
-    data.endPosition && data.startPosition !== data.endPosition
+  valueGetter: ({ data }) => data.chromosome && `${data.chromosome}:${data.startPosition}${data.endPosition && data.startPosition !== data.endPosition
```
Comment (areisle) [inline app/views/ReportView/components/SmallMutations/columnDefs.ts:20]: this also seems like unintended changes

File: app/views/ReportView/components/Pharmacogenomic/columnDefs.ts [hunk 3]
```diff
@@ -39,15 +39,11 @@ const columnDefs: ColDef[] = [{
       return `${variant.gene.name} ${variant.cnvState}`;
     }
     if (variantType === 'sv') {
-      return `(${
```
Comment (areisle) [inline app/views/ReportView/components/Pharmacogenomic/columnDefs.ts:42]: here as well

File: app/views/ReportView/components/CancerPredisposition/columnDefs.ts (hunk 1 above)
Comment (kttkjl) [reply app/views/ReportView/components/CancerPredisposition/columnDefs.ts:42]: yeah autofixed

File: app/views/ReportView/components/SmallMutations/columnDefs.ts (hunk 2 above)
Comment (kttkjl) [reply app/views/ReportView/components/SmallMutations/columnDefs.ts:20]: yep ... auto fix does funny things ...

File: app/views/ReportView/components/Pharmacogenomic/columnDefs.ts (hunk 3 above)
Comment (kttkjl) [reply app/views/ReportView/components/Pharmacogenomic/columnDefs.ts:42]: Honestly think the original looks clearer, but if autofix decides that it should change ...

**Review by areisle — APPROVED** [review]
